docs/widget_gallery.md

```diff
@@ -9,6 +9,11 @@ Welcome to the Textual widget gallery.
 
 We have many more widgets planned, or you can [build your own](./guide/widgets.md).
 
+Most of the code demonstrating each widget is fully working&mdash;you could cut and paste it if you wanted to.
```

Comment:
I don't think the repetition is necessary.

Reply:
The repetition is only found if you happen to follow a particular path through the documentation.   As people use different paths, it is very possible that the widget reference will be the only time they see the comment.  While code avoids repetition for correctness, documentation embraces it for effectiveness.